Cargo.toml

```
@@ -47,6 +47,7 @@ once_cell = "1.17.0"
 owo-colors = { version = "3.5.0", features = ["supports-colors"] }
 rayon = "1.6.1"
 regex = "1.7.1"
+reqwest = { version = "0.11.14", features = ["blocking"] }
```

```suggestion
reqwest = { version = "0.11.14", features = ["blocking"] }
openssl = { version = "0.10", features = ["vendored"] }
```

I think this will fix the build

---

agree on keeping it since https is pretty much standard these days